src/getInitialSettings.js

```diff
@@ -42,7 +42,7 @@ export function getInitialSettings() {
     settings.transmitInterval = +get('data-interval') || 5000;
     settings.logCountThreshold = +get('data-threshold') || 5;
     settings.userId = get('data-user') || null;
-    settings.version = get('data-version') || null;
+    settings.toolVersion = get('data-version') || null;
```

Comment:
this is where I was referencing changing the script tag to match the setting name.

so it'd be:
`get('data-tool-version')`

This is a breaking change, because it changes the script-tag api. And since toolVersion is required, the script would throw if users did not set this field.

We should add a default besides `null` to keep it from breaking (e.g. '1.0').